unit10/main.js

```diff
@@ -0,0 +1,138 @@
+// Bài 1: In một bảng nhân của 1 số với các số từ 1-> 10 và hiển thị kết quả:	
+function createMulTab(number) {
+  if (typeof(number) !== 'number') {
+    console.log ('Du lieu khong hop le')
+  } else {
+    for (let i = 1; i <= 10; i++) {
+      console.log ('  ' + number +  ' x ' + i + ' = ' + (number*i))
+    }
+  }
+}
+
+
+// Bài 2: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n:
+const getEvenNumber = n => {
+  if (typeof(n) !== 'number' || n < 1 || n > 30) {
+    console.log('Du lieu khong hop le')
+  } else {
+    for (let i = 2; i <= n; i++) {
+      if ( !(i % 2) ) {
+        console.log(i)
+      }
+    }
+  }
+}
+
+
+// Bài 3: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n:
+const getSum = n => {
+  let sum = 0  
+  if (typeof(n) !== 'number' || n < 1 || n > 30) {
+    console.log('Du lieu khong hop le')
+  } else {
+    for (let i = 1; i <= n; i++) {
+      sum +=i
+    }
+  }
+  return sum
+}
+
+
+// Bài 4: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In ra n! (giai thừa của n):
+const getFactorial = n => {
+  let fac = 1
+  if (typeof(n) !== 'number' || n < 1 || n > 30) {
+    console.log('Du lieu khong hop le')
+  } else {
+    for (let i = 1; i <= n; i++) {
+      fac = fac*i
+    }
+  }
+  return fac
+}
+
+
+// Bài 5: Hàm nhận vào 1 mảng đếm xem trong mảng có bao nhiêu số chẵn:
+const getCountEven = arr => {
+  let count = 0
+  for (let i = 0; i < arr.length; i++) {
+    if ( !( arr[i] % 2 ))
+    count++
+  }
+  return count
+}
+
+// Bài 6: Hàm nhận vào 1 mảng chứa các chữ cái bị trùng lặp ngẫu nhiên Tính toán và trả về 1 mảng đã xóa đi các chữ cái bị trùng lặp:
+const deleteDup = arr => {
+  let newArr = []
```

Comment:
```suggestion
  const noDublicateArr = []
```
Hãy khai báo là `const` và đặt tên rõ nghĩa 1. tí